README.md

```diff
@@ -51,15 +51,23 @@ The files below make up the table of contents for a release distribution archive
 - `license-LGPL.txt` GNU Lesser General Public License 2.1
 - `javadoc/` Javadoc documentation
 - `lib/` JGraphT libraries and dependencies:
-    - `antlr4-runtime-x.y.jar` ANTLR parser runtime
-    - `commons-lang3-x.y.jar` Apache Commons Lang library
     - `jgrapht-core-x.y.z.jar` core library
     - `jgrapht-demo-x.y.z.jar` demo classes
     - `jgrapht-ext-x.y.z.jar` extensions
     - `jgrapht-ext-x.y.z-uber.jar` all libraries rolled into one
     - `jgrapht-io-x.y.z.jar` Importers/Exporters for various graph formats
     - `jgrapht-io-x.y.z-uber.jar` all libraries rolled into one
+    - `jgrapht-guava-x.y.z.jar` Adapter classes for the Guava library
+    - `jgrapht-guava-x.y.z-uber.jar` all libraries rolled into one
     - `jgraphx-a.b.c.jar` JGraphX dependency library
+    - `antlr4-runtime-x.y.jar` ANTLR parser runtime
+    - `commons-lang3-x.y.jar` Apache Commons Lang library
+    - `guava-x.y-jre.jar` Guava library
+    - `animal-shiffer-annotations-x.y.jar` Guava dependency
```

Comment:
Heh, it's actually animal-sniffer.  (Ew!)  And somehow it turned into animal-shifter further down...lycanthropy perhaps?

Reply:
Also, I'm wondering whether we can actually exclude all the annotations dependencies to keep things more streamlined.  It wasn't clear from here:

https://github.com/google/guava/issues/2824